projects/storefrontlib/src/lib/cms/store/selectors/component.selectors.ts

```diff
@@ -11,20 +11,17 @@ export const getComponentState: MemoizedSelector<
   (state: fromFeature.CmsState) => state.component
 );
 
-export const getComponentEntities: MemoizedSelector<any, any> = createSelector(
-  getComponentState,
-  fromComponent.getComponentEntities
-);
+export const getComponentEntities: MemoizedSelector<
+  any,
+  { [id: string]: any }
+> = createSelector(getComponentState, fromComponent.getComponentEntities);
 
 export const componentSelectorFactory = (uid): MemoizedSelector<any, any> => {
```

Comment:
`any` can be `T`.

Reply:
Same as above. Don't know how to implement

Reply:
Sorry, it should be `any`, because cms components can be any types.

Reply:
No worries :)